mongo/integration/gridfs_test.go

```diff
@@ -369,6 +370,40 @@ func TestGridFS(x *testing.T) {
 			_, err = bucket.OpenDownloadStream(oid)
 			assert.Equal(mt, gridfs.ErrMissingChunkSize, err, "expected error %v, got %v", gridfs.ErrMissingChunkSize, err)
 		})
+		mt.Run("cursor error during read after downloading", func(mt *mtest.T) {
+			// To simulate a cursor error we upload a file larger than the 16MB default batch size,
+			// so Read() will actually execute a getMore against the server. Since the ReadDeadline is
+			// set in the past, this should cause a timeout.
+
+			// Create file to upload.
+			fileName := "read-error-test"
+			fileData := make([]byte, 17000000)
+
+			// Create bucket.
+			bucket, err := gridfs.NewBucket(mt.DB)
+			assert.Nil(mt, err, "NewBucket error: %v", err)
+			defer func() { _ = bucket.Drop() }()
```

Comment:
What's the purpose of the ` _ = ` in this `bucket.Drop()` statement? I see it in other tests, too.

Reply:
`bucket.Drop()` drops the "files" and "chunks" collections associated with the bucket. In testing code, we generally explicitly ignore the potential error in a deferred collection drop (or client disconnect, etc.) with `_ =`.  I suppose it's clearer than just `bucket.Drop()`, and a potential error from collection dropping is of little relevance to the test. Anyone else have context on this, though?

Reply:
It's to prevent errors from `errcheck`. The other option is to modify the [errcheck-excludes](https://github.com/mongodb/mongo-go-driver/blob/master/.errcheck-excludes) file to indicate that errors from `Drop` can safely be ignored, but that takes effect globally, so we try not to use it too often.